SlackAPI/SlackClientBase.cs

```diff
@@ -110,7 +110,7 @@ protected HttpResponseMessage PostRequest(string requestUri, MultipartFormDataCo
             return httpClient.PostAsync(requestUri, form).Result;
         }
 
-        public void RegisterConverter(JsonConverter converter)
+        public void RegisterConverter(Newtonsoft.Json.JsonConverter converter)
```

Comment:
Why was this changed to `Newtonsoft.Json.JsonConverter`? It has the using at the top of the file?